### PR TITLE
Button Page

### PR DESCRIPTION
**Pull request by RakshithBhat03**
Added Button Page for documentation.

Preview: https://deploy-preview-16--clutchui.netlify.app/docs/button/button

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
The code looks clean and well structured. Good to merge

File: docs/button/button.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible"
+        content="IE=edge" />
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/f6863db2d0.js"
+        crossorigin="anonymous"></script>
```
Comment (nareshpoonia) [inline docs/button/button.html]: You can consider moving font-awesome link to your Main CSS, so you don't have to add this to all of your html files

**Review by nareshpoonia — APPROVED** [review]
Just a small suggestion on moving font-awesome link, other than that the code looks clean.
Good to merge

File: docs/button/button.html (hunk 1 above)
Comment (RakshithBhat03) [reply docs/button/button.html]: Thanks. Will do it.

**Comment by RakshithBhat03** [comment]
Implemented changes suggested by @nareshpoonia.

**Pull request merged**